microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,636 @@
+* [ICoreWebView2Environment5 interface](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment5#add_browserprocessexited)
+   * [ICoreWebView2Environment5::add_BrowserProcessExited event](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment5#add_browserprocessexited)
+   * [ICoreWebView2Environment5::remove_BrowserProcessExited event](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment5#remove_browserprocessexited)
+* [ICoreWebView2Environment8 interface](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment8)
+   * [ICoreWebView2Environment8::add_ProcessInfosChanged event](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment8#add_processinfoschanged)
+   * [ICoreWebView2Environment8::remove_ProcessInfosChanged event](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment8#remove_processinfoschanged)
+   * [ICoreWebView2Environment8::GetProcessInfos method](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment8#getprocessinfos)
+
+---
+
+This is an overview at the conceptual level. For detailed API names and related types in your app's framework, see [Process management](overview-features-apis.md#process-management) in _Overview of WebView2 features and APIs_.
+
+In this document, we discuss how to handle process exits and failures, using the `CoreWebView2.ProcessFailed` and `CoreWebView2Environment.BrowserProcessExited` events.
+
+
+<!-- ====================================================================== -->
+## Events for processes that exited or failed
+
+The WebView2 Runtime is composed of different types of processes, including:
+* Main browser process
+* Renderer processes
+* Utility processes
+* GPU process
+
+This is an illustrative and incomplete list of process kinds. The purpose and management details for these processes is part of the Chromium architecture and beyond the scope of this document, or the WebView2 architecture.
+
+When you create and initialize a WebView2 control, WebView2 will ensure there's a WebView2 Runtime to power your control and connect to its [WebView2 Process Group](process-model.md#processes-in-the-webview2-runtime). Once this connection is established, your control will start monitoring these processes for the following events:
+
+* **Main browser process exits.** If the main browser process exits for _any reason_, the `CoreWebView2Environment` will raise the `BrowserProcessExited` event. Use this event to synchronize operations involving the WebView2 Runtime resources and lifetime, such as _User Data Folder_ management and updates. See [Handle main browser process exited](#handle-main-browser-process-exited) below.
```

Comment:
I'm inclined to keep.
I now can view rendered page, to better evaluate the doc design:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-related-events?branch=pr-en-us-2670&tabs=dotnetcsharp#events-for-processes-that-exited-or-failed

Other comparable sections with similar nav? tbd

Reply:
@lflores-ms - i wasn't sure which section this would link to, so i added a hidden comment; pls find "todo" in file after i push commit.
```
* **Main browser process crashes.**  When the main browser process crashes, it will produce both a `ProcessFailed` event and a `BrowserProcessExited` event, since the main browser process _exited_ because of a failure.  <!--todo: link to a section below?-->
```